packages/mesos/build

```diff
@@ -17,6 +17,7 @@ LDFLAGS="-Wl,-rpath,/opt/mesosphere/lib" "/pkg/src/mesos/configure" \
   --with-libevent=/opt/mesosphere/active/libevent \
   --with-curl=/opt/mesosphere/active/curl \
   --with-boost=/opt/mesosphere/active/boost-libs \
+  --disable-werror \
```

Comment:
FYI this option does not cascade to `3rdparty/grpc-1.4.2`, presumably because that package doesn't support such an option.

Reply:
This is true  -  actually even worse; we blindly forward all kinds of mesos specific configure phase switches to 3rd party projects - lucky us this doesn't cause hard failures.